### DISCUSSION TITLE
blog/nextjs/1-ssr-ssg-isr

### DISCUSSION THREAD

**Discussion by giscus** [pySoo/Next-blog#75] [General]
# blog/nextjs/1-ssr-ssg-isr

Next.js의 렌더링 방식의 차이점에 대해서 알아보고, 필요에 따라 적절한 렌더링 기법을 적용하는 법에 대해 배워보겠습니다.

https://enjoydev.life/blog/nextjs/1-ssr-ssg-isr

**Comment by westofsky** [comment]
안녕하세요 블로그 잘 봤습니다! 저도 next로 이러한 블로그를 만들고 싶은데 혹시 참고할만한 사이트가 있을까요 ?? 직접 다 개발하신걸까요 ??

**Reply by pySoo** [reply]
안녕하세요! 방문해주셔서 감사합니다 😀
저는 [bepyan님의 블로그 만들기 가이드 글](https://bepyan.github.io/blog/nextjs-blog)과 코드를 참고해서 블로그를 만들었어요!

기능과 관련된 코드는 최대한 리팩토링하고 제가 필요하다고 생각한 기능들을 추가하는 형식으로 만들어보았습니다 ㅎㅎ
예를 들면 예시 블로그에서는 포스트 검색을 블로그 탭에서만 할 수 있는데
저는 모든 탭에서 검색하게 해주고 싶어서 포스트 검색창을 헤더로 분리하는 작업을 하였어요!

혹시 도움이 될까 싶어서 [제 프로젝트 코드](https://github.com/pySoo/Next-blog)도 남겨보겠습니다 ㅎㅎ

**Comment by im-dullin** [comment]
정말 가독성도 좋고 이해가 잘 가는 글이였습니다.

덕분에 개념정리가 훨씬 쉬웠고, 블로그부터 깃허브
활동하신 글을 모두 보았는데 확실한 동기부여를 얻고 갑니다ㅎㅎ

감사합니다!

**Reply by pySoo** [reply]
헉 좋은 말씀 너무 감사드립니다 ☺️
이런 칭찬을 듣게 되다니..! 블로그 운영하는 보람이 느껴지네요.. 👍
앞으로도 꾸준히 의미있는 글들 작성해보도록 하겠습니다 ㅎㅎ 방문해주셔서 감사합니다!

**Comment by SeoHee3478** [comment]
글이 너무 잘 익혀서 개념을 이해하는데 큰 도움이 되었습니다. 좋은 글 감사합니다 !

**Reply by pySoo** [reply]
안녕하세요 서희님! 도움이 되었다니 정말 기쁘네요 😊
좋은 말씀 너무 감사드립니다 ㅎㅎ 방문해주셔서 감사해요!

**Comment by osh6006** [comment]
진짜 완벽하게 이해가 갔습니다 존경합니다 저도 Next.js 렌더링 관련 글을 적고 있는데 이렇게 정리가 잘 된 글은 처음 보네요

**Reply by pySoo** [reply]
헉 과찬이십니다...! 도움이 되었다니 다행입니다 🙇‍♀️
요즘 글을 자주 못쓰고 있는데 부지런히 새로운 글을 써봐야겠다는 동기부여가 되네요 ㅎㅎ 감사합니다!!!

**Comment by osh6006** [comment]
아 그리고 질문이 있는데 혹시 글을 작성하실 때 

인라인코드 블럭을 쓰는 기준을 알 수 있을까요?? 

어떤 곳은 `인라인 코드블럭`을 사용하고 있고 어떤 곳은 **볼드**로 강조하고 있으셔서 그 차이를 알고 싶습니다!

**Reply by pySoo** [reply]
개인적으로는 코드블럭이 눈에 더 잘 들어오는 느낌이어서
**볼드**보다 더 중요도가 높은 개념은 **`코드블럭`** 으로 강조하고 있습니다!

혹시 강조하는 부분은 통일해서 표시하는게 읽으실 때 더 편하실까요?!
의견 주시면 반영해보도록 하겠습니다!! 👍

**Reply by osh6006** [reply]
아 아닙니다! 그냥 저도 **볼드**랑 `코드블럭`을 섞어서 글을 쓰곤 하는데,

그 기준이 명확하지 않아서 어떤 부분에서 그렇게 사용하셨는지 궁금했습니다!

글의 본문으로 들어가기 전에,

 **볼드** 보다 `코드블럭`이 더 중요한 개념이라고 설명 해주시면  처음 글을 읽을 때 더 이해가 잘될 것 같습니다!

사실 안 하셔도 이해가 너무 잘 되어서 다른 글들도 읽어보고 있습니다!

**Reply by pySoo** [reply]
오오 좋은 의견 감사드립니다!! 🙇‍♀️
저도 마크다운을 어떻게 활용하면 더 가독성이 좋아질지 항상 고민하는 것 같아요..!

아무래도 강조되는 부분이 여러 곳이다보니
더 우선순위가 높은 개념에 집중하면 이해에 더 도움이 될 것 같습니다!
의견 감사합니다 ㅎㅎ 🚀👍
